extension/src/main/java/com/cliqz/extension/CliqzExtensionFeature.kt

```diff
@@ -0,0 +1,142 @@
+package com.cliqz.extension
+
+import kotlinx.coroutines.*
+import mozilla.components.browser.engine.gecko.webextension.GeckoWebExtension
+import mozilla.components.concept.engine.EngineSession
+import mozilla.components.concept.engine.webextension.MessageHandler
+import mozilla.components.concept.engine.webextension.Port
+import mozilla.components.support.base.log.logger.Logger
+import org.json.JSONArray
+import org.json.JSONObject
+import org.mozilla.geckoview.GeckoRuntime
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.CountDownLatch
+
+val DEMOGRAPHICS = mapOf(
+        "brand" to "cliqz",
+        "name" to "browser",
+        "platform" to "android"
+)
+val TELEMETRY_CHANNEL = if (BuildConfig.DEBUG) "MA99" else "MA60"
```

Comment:
this is legacy telemety setup which should be killed by now

Reply:
please check with @remusao @cliqz/dominik-s-team

Reply:
The main side effect of this setting is that anolysis will use staging when the channel is "MA99": https://github.com/cliqz/navigation-extension/blob/master/modules/anolysis/sources/cliqz-config.es#L23-L30

Reply:
Best to do it by passing `debug` flag to App constructor or to set developer pref.

Reply:
As I'm already passing the `debug` flag then I guess this is redundant then. Will remove

https://github.com/cliqz/daisy/pull/52/files#diff-2ef0ecff1288fe203f1540106d28088eR99